### PR TITLE
Add phone number, refactor address form

### PR DESCRIPTION
**Pull request by ds300**
- bumped metaphysics to include changes from artsy/metaphysics#1265
- Added phone number field to AddressForm, while refactoring it to not have individual change handlers for each field, as discussed in https://github.com/artsy/reaction/pull/1152#pullrequestreview-147370701 and https://github.com/artsy/reaction/pull/1186#discussion_r213277528

https://artsyproduct.atlassian.net/browse/PURCHASE-422
https://artsyproduct.atlassian.net/browse/PURCHASE-397

force screenshots of phone number appearing in:
- review address
    image
- summary address
    image
- shipping address form
    image

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 53c3de65cf8feaa1b8bebaf7000ba9fd939db54d

https://deploy-preview-1217--artsy-reaction.netlify.com

File: src/Apps/Order/Components/AddressForm.tsx [hunk 1]
```diff
@@ -5,26 +5,70 @@ import { CountrySelect } from "Styleguide/Components"
...
+  postalCode: string
+  addressLine1: string
+  addressLine2: string
+  city: string
+  region: string
+  phoneNumber: string
 }
 
-export class AddressForm extends React.Component<AddressFormProps> {
+export const emptyAddress: Address = Object.freeze({
```
Comment (zephraph) [inline src/Apps/Order/Components/AddressForm.tsx]: Are we polyfilling Object.freeze? I remember seeing Core in our bundle payloads, so I think we probably are, but it'd be good to double check.

Comment (ds300) [reply src/Apps/Order/Components/AddressForm.tsx]: 🤔 do we need to? 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/freeze#Browser_compatibility

**Comment by ds300** [comment]
Alright this is 🍏

File: src/Apps/Order/Components/AddressForm.tsx (hunk 1 above)
Comment (zephraph) [reply src/Apps/Order/Components/AddressForm.tsx]: You make a valid point 😄

File: src/Apps/Order/Routes/Payment/index.tsx [hunk 2]
```diff
@@ -14,17 +14,20 @@ import { Col, Row } from "Styleguide/Elements/Grid"
...
 import { TransactionSummaryFragmentContainer as TransactionSummary } from "../../Components/TransactionSummary"
-import { Address } from "../Shipping"
 
 export interface PaymentProps {
   order: Payment_order
 }
 
 interface PaymentState {
-  country: string
+  address: Partial<Address>
```
Comment (ds300) [inline src/Apps/Order/Routes/Payment/index.tsx]: argh, this should not be `Partial`

note to self: fix

File: src/Apps/Order/Components/AddressForm.tsx [hunk 3]
```diff
@@ -5,26 +5,72 @@ import { CountrySelect } from "Styleguide/Components"
...
+  AddressFormState
+> {
+  state = {
+    address: { ...emptyAddress, ...this.props.defaultValue },
+  }
+
+  changeEventHandler = (key: keyof Address) => (
+    ev: React.FormEvent<HTMLInputElement>
+  ) => {
+    console.log("FECK", ev.currentTarget.value)
```
Comment (ashfurrow) [inline src/Apps/Order/Components/AddressForm.tsx]: Ah, missed this.

Comment (ds300) [reply src/Apps/Order/Components/AddressForm.tsx]: :o 

Fixed in #1222

**Pull request merged**